### DISCUSSION TITLE
Why entire Load is getting reflected on one node while other node is idle in Ant Media Server Cluster?

### DISCUSSION THREAD

**Discussion by asharamseervi** [ant-media/Ant-Media-Server#4985] [FAQs]
I have created an Ant Media Server Cluster using AWS CloudFormation and when I'm publishing/playing the streams all load is getting reflected on one node while other nodes are idle.

**Comment by USAMAWIZARD** [comment]
It seems like you are publishing &amp; playing from the same node in cluster due to which all traffic is reflecting on a single node.
Please make sure that you are using origin nodes for publishing and edge nodes for playing streams. URLs should be like below:
For Origin: <https://domain-name/>
For Edge : <https://domain-name:5443/>
